src/chatbot/chatbot.clj

```diff
@@ -133,6 +133,15 @@
     }
 )
 
+(
+        defn get-park-titles-list
```

Comment:
Function has to be applied to the functions that require the set of park titles.